index.js

```diff
@@ -1,93 +1,59 @@
-}
-
-function deleteFromLocalStorage(id) {
-  let arr = localStorage.getItem('pdfs') ? JSON.parse(localStorage.getItem('pdfs')) : [];
-
-  arr = arr.filter((textbk) => {
-    if (textbk.id === id) {
-      return textbk;
-    }
-    return false;
-  });
-  localStorage.setItem('pdfs', JSON.stringify(arr));
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
 }
 
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
-
-  if (title.value && author.value) {
-    const id = new Date().getTime().toString();
-    const book = document.createElement('div');
-    book.className = 'the-book';
-    const attribute = document.createAttribute('data-id');
-    attribute.value = id;
-    book.setAttributeNode(attribute);
-    book.innerHTML = ` <ul class="list-unstyled">
-       <li class="titles">${title.value}</li>
-       <li class="book-author">${author.value}</li>
-   </ul>
-   <button class="btn-remove">Remove</button>`;
-    const pdfs = {};
-    pdfs.title = title.value;
-    pdfs.author = author.value;
-
-    container.appendChild(book);
-    title.value = '';
-    author.value = '';
+let booksList = JSON.parse(localStorage.getItem('book')) || [];
 
-    const removeBtn = book.querySelector('.btn-remove');
-    removeBtn.addEventListener('click', (e) => {
-      e.preventDefault();
+const books = document.querySelector('.books');
 
-      const book = e.target.parentElement;
-
-      const id = book.dataset;
-      container.removeChild(book);
-      deleteFromLocalStorage(id);
-    });
-
-    saveLocalStorage(pdfs, id);
+// function to get books from localStorage
+const getBooks = () => {
+  const storageData = localStorage.getItem('books');
+  if (storageData === undefined || storageData === null) {
+    return;
   }
+  const storageBooks = JSON.parse(storageData);
+  booksList = storageBooks;
+  books.innerHTML = storageBooks
+    .map(
+      (book, index) => `<div class=" highlight ${index % 2 === 0 ? 'highlight-bg' : ''}">
+                    <div>
+                      "${book.title}" by ${book.author}
+                      <button type="button" onclick='removeBook(${book.id})'>Remove</button>
+                    </div>
+                </div>`,
+    )
+    .join('');
+};
+
+// remove book from books collections
+const removeBook = (id) => {
+  if (id === null) return;
+  const newBooks = booksList.filter((book) => book.id !== id);
+  localStorage.setItem('books', JSON.stringify(newBooks));
+  getBooks();
+};
+
+// to add book from books collection
+const booksForm = document.querySelector('.new-book-form');
+booksForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const id = Math.floor(Math.random() * 10000); // Asign random Id for each
+  const item = new Book(id, title, author); // Add book for the class Book
```

Comment:
Kindly check the add functionality, I tried adding a book and author but it's not adding. Kindly make the necessary changes as this is part of the core functionalities of your project.⚔️🟢